### ISSUE TITLE
arm version ?

### ISSUE THREAD

**Issue by nsteinmetz** [cozy/cozy-debian#25]
Hi,

Is an arm version of packages planned to install cozy on arm boards such as cubietruck in my case ?

Thanks,
Nicolas

**Comment by clochix** [comment]
Hi @nsteinmetz,

nice to see you here ;-)

We use to have an image for Cubietruck, but [we removed it](https://github.com/cozy/cozy-docs/commit/6c2ca444538e9bfba2724907f4cbe2eeac3c45fc#diff-d5f8969c4ea5107afe6aadc7bef1233a) as we have no time to update it. But I think you could easily install Cozy on your board, following our [step by step guide](https://docs.cozy.io/en/host/install/install-step-by-step.html). Don't hesitate to come to IRC to ask for help

For now, we are working on a new image for the Pine64 board.

**Comment by nsteinmetz** [comment]
Hi @clochix 

Thanks for the warm welcome ;-)

I knew there were one cubietruck image in the past ; I followed the step by step guide as you could see with https://github.com/cozy/cozy-docs/issues/265 & https://github.com/cozy/cozy-controller/issues/107.

Let's see if I can achieve the step by step guide then and hopefully the `arm64` image for Pine will allow a `arm` build too ;-)

**Comment by nledez** [comment]
Hi @nsteinmetz,

Can you give me more information about your Linux?

`lsb_release -a ; uname -a`

Thanks.

**Comment by nsteinmetz** [comment]
Hi @nledez 

Below the answer:

```
root@cubie1:~# lsb_release -a ; uname -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 8.3 (jessie)
Release:    8.3
Codename:   jessie
Linux cubie1 4.4.1-sunxi #6 SMP Thu Feb 11 21:57:14 CET 2016 armv7l GNU/Linux
```

Just saw via the [tuto on rpi2](https://forum.cozy.io/t/tuto-cozy-raspberry-pi-2-application-android/2002) that you have an `armhf` repo but it lacks `couchdb` package:

```
root@cubie1:~# apt install cozy
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 cozy : PreDepends: couchdb but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by nledez** [comment]
Hi all,

Thanks for response @nsteinmetz :)

Cozy build the CouchDB only for amd64. The armhf is provided by Raspbian.

We not planned to build CouchDB for other architecture.

To explain why, in the past I write this in french:
https://forum.cozy.io/t/packaging-debian-ubuntu/1082

**Comment by RobYed** [comment]
For anyone, who is still interested in running cozy on a Cubietruck with Docker:

You only need to replace the base image of the Dockerfile from
`ubuntu:14.04` to `armv7/armhf-ubuntu:14.04` and then build the docker container. I simply forked cozy-docker and edited the Dockerfile. So I was able to build with this command:

`sudo docker build -t cozy/full github.com/RobYed/cozy-docker`

I tested it and it works so far, although I have to admit that some errors occured while the cozy apps were installed. I have not yet found out whether these errors influence the usage of cozy.
